akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -280,120 +261,95 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
+  val `application/vnd.openxmlformats-officedocument.presentationml.template`     = abin("vnd.openxmlformats-officedocument.presentationml.template", compressible, "potx")
+  val `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`         = abin("vnd.openxmlformats-officedocument.spreadsheetml.sheet", compressible, "xlsx")
+  val `application/vnd.openxmlformats-officedocument.spreadsheetml.template`      = abin("vnd.openxmlformats-officedocument.spreadsheetml.template", compressible, "xltx")
+  val `application/vnd.openxmlformats-officedocument.wordprocessingml.document`   = abin("vnd.openxmlformats-officedocument.wordprocessingml.document", compressible, "docx")
+  val `application/vnd.openxmlformats-officedocument.wordprocessingml.template`   = abin("vnd.openxmlformats-officedocument.wordprocessingml.template", compressible, "dotx")
+  val `application/x-7z-compressed`                                               = abin("x-7z-compressed", uncompressible, "7z", "s7z")
+  val `application/x-ace-compressed`                                              = abin("x-ace-compressed", uncompressible, "ace")
+  val `application/x-apple-diskimage`                                             = abin("x-apple-diskimage", uncompressible, "dmg")
+  val `application/x-arc-compressed`                                              = abin("x-arc-compressed", uncompressible, "arc")
+  val `application/x-bzip`                                                        = abin("x-bzip", uncompressible, "bz")
+  val `application/x-bzip2`                                                       = abin("x-bzip2", uncompressible, "boz", "bz2")
+  val `application/x-chrome-extension`                                            = abin("x-chrome-extension", uncompressible, "crx")
+  val `application/x-compress`                                                    = abin("x-compress", uncompressible, "z")
+  val `application/x-compressed`                                                  = abin("x-compressed", uncompressible, "gz")
+  val `application/x-debian-package`                                              = abin("x-debian-package", compressible, "deb")
+  val `application/x-dvi`                                                         = abin("x-dvi", compressible, "dvi")
+  val `application/x-font-truetype`                                               = abin("x-font-truetype", compressible, "ttf")
+  val `application/x-font-opentype`                                               = abin("x-font-opentype", compressible, "otf")
+  val `application/x-gtar`                                                        = abin("x-gtar", uncompressible, "gtar")
+  val `application/x-gzip`                                                        = abin("x-gzip", uncompressible, "gzip")
+  val `application/x-latex`                                                       = abin("x-latex", compressible, "latex", "ltx")
```

Comment:
Latex consists of plain text and do not contain hidden formatting codes or binary instructions, so this should be awoc?